message.go

```diff
@@ -249,6 +249,10 @@ type MessageEdit struct {
 	Embeds          []*MessageEmbed         `json:"embeds"`
 	AllowedMentions *MessageAllowedMentions `json:"allowed_mentions,omitempty"`
 	Flags           MessageFlags            `json:"flags,omitempty"`
+	// Files to append to the message
+	Files []*File `json:"files,omitempty"`
```

Comment:
Files are sent using `MultipartBodyWithJSON`, therefore should be omitted from the request.
```suggestion
	Files []*File `json:"-"`
```